ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -33,3 +33,7 @@ tests:
   commands: make verify-all
   container:
     from: src
+- as: e2e-wmcb
```

Comment:
Perhaps something to signify `ci`?

Reply:
Wouldn't that be stating the obvious? I prefer seeing `ci/prow/e2e-wmcb`. This is a good pattern and is being followed in other repos.